app/assets/javascripts/discourse/lib/search.js.es6

```diff
@@ -94,9 +94,9 @@ export function searchForTerm(term, opts) {
     };
   }
 
-  var promise = ajax('/search/query', { data: data });
+  let promise = ajax('/search/query', { data: data });
```

Comment:
Sorry, I'm still new to es6, is there any reason not to use a `const` here ?

Reply:
😀😆👍🥰🥰🧗‍♂️🏇🏂